src/js/pannellum.js

```diff
@@ -605,7 +608,14 @@ function parseGPanoXMP(image, url) {
                         config.horizonRoll = xmp.horizonRoll;
                 }
                 
-                // TODO: add support for initial view settings
+                if (xml.pitch != null && xml.yaw != null && xml.hfov != null) {
```

Comment:
This should be `xmp`. Also, I see no reason why all three should need to be defined; some of the other parameters are grouped because they interact with each other, but these parameter do not. I'd use something like this for each of the three parameters:
```javascript
if (xmp.pitch !== null && specifiedPhotoSphereExcludes.indexOf('pitch') < 0)
    config.pitch = xmp.pitch;
```

Reply:
typo fixed in forced push. it's doesnt make sense, if someone sets only one parameter (pitch or yaw or hfov). But if you like, i can split it.